.travis.yml

```diff
@@ -44,9 +44,7 @@ jobs:
       before_install:
         - sudo apt-get install pandoc
```

Comment:
There is a better way of doing this: https://docs.travis-ci.com/user/installing-dependencies/#adding-apt-packages